### PR TITLE
feat(LineFollower): add line follower

### LINKED ISSUE #8: Autonomous line follower
Make some kind of autonomous line follower that could follow a colored line.

### PR DESCRIPTION
**Pull request by ariporad**
This is untested, but should work. Feel free to review the code anyway. I'll test it on Sunday.

Fixes #8.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/debug/LineFollower.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	private com.qualcomm.robotcore.hardware.LightSensor lightSensor;
+	private DcMotor leftMotor;
+	private DcMotor rightMotor;
+
+	private org.redshiftrobotics.linefollower.LineFollower lineFollower;
+
+	@Override
+    public void init() {
+        telemetry.addData("Status", "Initialized");
+
```
Comment (NoahRoseLedesma) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/debug/LineFollower.java]: Fix uneven spacing

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/debug/LineFollower.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+
+@Autonomous(name="Line Follower", group="Debug")
+public class LineFollower extends OpMode
+{
+	private com.qualcomm.robotcore.hardware.LightSensor lightSensor;
+	private DcMotor leftMotor;
+	private DcMotor rightMotor;
+
+	private org.redshiftrobotics.linefollower.LineFollower lineFollower;
```
Comment (NoahRoseLedesma) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/debug/LineFollower.java]: Import  `org.redshiftrobotics.linefollower.LineFollower` to address it by it's short name.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/debug/LineFollower.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+
+@Autonomous(name="Line Follower", group="Debug")
+public class LineFollower extends OpMode
+{
+	private com.qualcomm.robotcore.hardware.LightSensor lightSensor;
```
Comment (NoahRoseLedesma) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/debug/LineFollower.java]: Import `com.qualcomm.robotcore.hardware.LightSensor` to address it by it's short name

File: TeamCode/src/main/java/org/redshiftrobotics/linefollower/LineFollower.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	private double ON_LINE_LEFT_POWER;
+	private double ON_LINE_RIGHT_POWER;
+	private double OFF_LINE_LEFT_POWER;
+	private double OFF_LINE_RIGHT_POWER;
+
+	public LineFollower(LightSensor lightSensor, DcMotor leftMotor, DcMotor rightMotor) {
+		new LineFollower(lightSensor, leftMotor, rightMotor, 0.1, 1, 0.9, 0.5, 1);
+	}
+
+	public LineFollower(
```
Comment (NoahRoseLedesma) [inline TeamCode/src/main/java/org/redshiftrobotics/linefollower/LineFollower.java]: Can you add JavaDoc for the use of this constructor's parameters.

**Comment by ariporad** [comment]
Reviewed in person -- merging.

**Pull request merged**